### PR TITLE
Unlock sneakers adapter version

### PR DESCRIPTION
**Pull request by artofhuman**

### CODE REVIEW

**Comment by matthewd** [comment]
If we no longer need a prerelease, I guess we should follow the other adapters lead, and drop the version constraint entirely.

Before we deal with that, however, we probably need to fix the test failure. :smile:

**Comment by artofhuman** [comment]
Ok, i try to fix tests

**Comment by artofhuman** [comment]
AJ section is green https://travis-ci.org/rails/rails/builds/60330170

File: activejob/test/support/integration/adapters/sneakers.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def bunny_publisher
     end
 
     def bunny_queue
-      @queue ||= bunny_publisher.exchange.channel.queue "integration_tests", durable: true
+      @queue ||= bunny_publisher.send(:exchange).channel.queue "integration_tests", durable: true
```
Comment (senny) [inline activejob/test/support/integration/adapters/sneakers.rb]: Looks like `exchange` has been removed from the public API. Do you think there's any chance that we can implement this without reaching into the object? If not, maybe we can persuade the sneakers maintainer to consider making it public again...

/cc @jondot

Comment (artofhuman) [inline activejob/test/support/integration/adapters/sneakers.rb]: Yes. exchange moved to private api in this commit https://github.com/jondot/sneakers/commit/c331d118f5b2f4cdb5c5f7a5d06cadf8cbc5f40e

Comment (jondot) [inline activejob/test/support/integration/adapters/sneakers.rb]: Hey guys, I can explore exposing the exchange back. Thanks for notifying me!
Will get back with updates soon

Comment (jondot) [inline activejob/test/support/integration/adapters/sneakers.rb]: Done. Exchange moved back, and released new v1.0.4 to reflect this (and other previous improvements).
Feel free to LMK if there's anything more I can do to help, thanks!

**Comment by senny** [comment]
@artofhuman thank you :yellow_heart:

**Comment by artofhuman** [comment]
@senny @jondot Thanks

**Pull request merged**